WeakAuras/Prototypes.lua

```diff
@@ -8563,6 +8563,14 @@ Private.event_prototypes = {
           return state.duration - state.stagesData[5].start, state.duration - state.stagesData[5].finish
         end,
         enable = WeakAuras.IsRetail
+      },
+      {
+        name = L["Empowered Fully Charged"],
+        func = function(trigger, state)
+          if not state.stageTotal or not trigger.use_showChargedDuration then return 0, 0 end
+          return state.duration - state.stagesData[state.stageTotal].finish, 0
+        end,
+        enable = WeakAuras.IsRetail
```

Comment:
missing () here

Reply:
changed for all overlays

Reply:
reverted because it break call to enable function

```
1x WeakAuras/GenericTrigger.lua:1272: attempt to call field 'enable' (a boolean value)
[string "@WeakAuras/GenericTrigger.lua"]:1272: in function `Add'
[string "@WeakAuras/WeakAuras.lua"]:2955: in function <WeakAuras/WeakAuras.lua:2878>
[string "@WeakAuras/WeakAuras.lua"]:3036: in function `Add'
[string "@WeakAuras/WeakAuras.lua"]:2504: in function `AddMany'
[string "@WeakAuras/WeakAuras.lua"]:1150: in function <WeakAuras/WeakAuras.lua:1121>

[string "@WeakAuras/WeakAuras.lua"]:1273: in function <WeakAuras/WeakAuras.lua:1224>
```